src/main/java/com/arcbees/plugin/idea/utils/PackageIndex.java

```diff
@@ -0,0 +1,57 @@
+/**
+ * Copyright 2013 ArcBees Inc.
```

Comment:
Yeah, I understand, no problem. We try to have the inception year of the class as the copyright.